alphagov/paas-bootstrap
# [#162448881] Enable Prometheus metrics for Concourse

# What

We enable the Prometheus metrics endpoint on Concourse. As Concourse only supports one metric emitter we have to disable the Riemann one which was sending the job metrics to DataDog.

We've forked concourse/atc and concourse/concourse-bosh-release and had to build our custom Bosh release to include the job name in the `concourse_builds_finished` metrics. See https://github.com/alphagov/paas-release-ci/pull/87 for details.

# How to review

1. Run the `create-bosh-concourse` pipeline from this branch:

```
ENABLE_DATADOG=true BRANCH=prometheus_concourse_162448881 make dev deployer-concourse pipelines
```

2. Check if Concourse is deployed successfully.
3. Continue with https://github.com/alphagov/paas-cf/pull/1681

# Who can review

Not me.
